File: eCopy.Model/Requests/CityRequest.cs

namespace eCopy.Model.Requests
{
    public class CityRequest
    {
        public string Name { get; set; }
        public string ShortName { get; set; }
        public int PostalCode { get; set; }
        public int CountryID { get; set; }
        public bool Active { get; set; }
    }
}

File: ecopymobile/myapp/lib/model/cityRequest.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'cityRequest.g.dart';

@JsonSerializable()
class CityRequest {
  String? name;
  String? shortName;
  int? postalCode;
  int? countryID;
  bool? active;

  CityRequest() {}

  factory CityRequest.fromJson(Map<String, dynamic> json) =>
      _$CityRequestFromJson(json);

  /// Connect the generated [_$CityRequestToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$CityRequestToJson(this);
}
